addons/survey/static/src/js/survey_form.js

```diff
@@ -46,6 +53,34 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
     // Handlers
     // -------------------------------------------------------------------------
 
+    _onKeyPress: function (event) {
+        var self = this;
+        var aIndex = 'a'.charCodeAt(0);
+        var zIndex = 'z'.charCodeAt(0);
+        var code = event.keyCode;
+        var insideTextArea = this.$("textarea").is(":focus");
```

Comment:
insideTextArea seems to be a precondition for both.

Maybe do:

```
if(this.$("textarea").is(":focus")) {
     return;
}
```

At the beginning to reduce the complexity?